src/components/SpecialResolution/HelpSpecialResolution.vue

```diff
@@ -0,0 +1,200 @@
+<template>
+  <div>
+    <div class="help-section-component ma-6">
+      <div
+        class="help-btn top"
+        @click="helpToggle = !helpToggle"
+      >
+        <v-icon color="primary">
+          mdi-help-circle-outline
+        </v-icon>
+        <span
+          v-if="!helpToggle"
+          class="pl-2"
+        >Help with Special Resolution</span>
+        <span
+          v-else
+          class="pl-2"
+        >Hide Help</span>
+      </div>
+      <v-expand-transition v-if="helpToggle">
+        <section class="help-section info-text">
+          <header class="help-header mt-4">
+            <h2 id="sr-heading">
+              Help with Special Resolution
+            </h2>
+          </header>
+          <br>
+          <div>
+            <p>
+              A special resolution is a resolution by the members of a Cooperative Association.
+              It is required if you wish to change the Cooperative Name, Type, Rules of Association
+              or Memorandum of Association.
+            </p>
+            <p>
+              A special resolution is submitted to a vote.
+              It is passed by either being consented to in writing by all members, or in a general meeting.
```

Comment:
Is this supposed to be 2 lines? (I don't see it in the InVision design.)

Reply:
I think it's two lines because I was hitting the text width limit, but looks like 1 line in the UI